docs/source-pytorch/model/manual_optimization.rst

```diff
@@ -98,6 +98,41 @@ after every ``N`` steps, you can do as such.
             opt.step()
             opt.zero_grad()
 
+Gradient Clipping
+=================
+
+You can clip optimizer gradients during manual optimization similar to passing the ``gradient_clip_val`` and
+``gradient_clip_algorithm`` argument in :ref:`Trainer <trainer>` during automatic optimization.
+To perform gradient clipping with one optimizer with manual optimization, you can do as such.
+
+.. testcode:: python
+
+    from pytorch_lightning import LightningModule
+
+
+    class SimpleModel(LightningModule):
+        def __init__(self, gradient_clip_val=0.5, gradient_clip_algorithm="norm"):
+            super().__init__()
+            self.automatic_optimization = False
+            self.gradient_clip_val = gradient_clip_val
+            self.gradient_clip_algorithm = gradient_clip_algorithm
+
+        def training_step(self, batch, batch_idx):
+            opt = self.optimizers()
+
+            # compute loss
+            loss = self.compute_loss(batch)
+
+            opt.zero_grad()
+            self.manual_backward(loss)
+
+            # clip gradients
+            self.clip_gradients(opt, self.gradient_clip_val, self.gradient_clip_algorithm)
```

Comment:
I would suggest simplifying the example by avoiding the instance attributes, which should make it a bit easier to read. Also including the keyword names for clarity

```suggestion
            self.clip_gradients(opt, gradient_clip_val=0.5, gradient_clip_algorithm="norm")
```

Same comment for the other files.

Reply:
Updated it in both files.